### ISSUE TITLE
Incorrect documentation. Module not found: Can't resolve 'react-draft-wysiwyg'

### ISSUE THREAD

**Issue by gutmann-dev** [jpuri/react-draft-wysiwyg#1069]
import { Editor } from "react-draft-wysiwyg";
import "react-draft-wysiwyg/dist/react-draft-wysiwyg.css";
<Editor
  editorState={editorState}
  toolbarClassName="toolbarClassName"
  wrapperClassName="wrapperClassName"
  editorClassName="editorClassName"
  onEditorStateChange={this.onEditorStateChange}
/>;

ITS NOT WORKING..
Module not found: Can't resolve 'react-draft-wysiwyg'

Authors of this library, Have you tried to install that yourself? Lol

**Comment by theexplay** [comment]
You must to install peer dependencies by yourself, read documentation

**Comment by baldemarch** [comment]
Documentation says
```
npm install --save react-draft-wysiwyg draft-js
```

However, after doing it and also checking package.json in which I have
```
"draft-js": "^0.11.7",
"react-draft-wysiwyg": "^1.14.5",
```

I still have the same error:

<img width="421" alt="image" src="https://user-images.githubusercontent.com/1851559/106547681-4bf29c00-64d3-11eb-9999-ede919eb68ac.png">

**Comment by theexplay** [comment]
Are you using typescript?
If so, try installing types https://www.npmjs.com/package/@types/react-draft-wysiwyg

Is there a mistake in ide or when building the project too?

**Comment by baldemarch** [comment]
@theexplay No, I'm not using typescript, I also did not have any errors while building the app or in my IDE; however, installing the types package did fix the issue, thanks!
